auto-value-moshi/src/test/java/com/ryanharter/auto/value/moshi/AutoValueMoshiExtensionTest.java

```diff
@@ -86,8 +69,8 @@
         + "}\n"
     );
 
-    JavaFileObject expected = JavaFileObjects.forSourceString("test/AutoValue_Test", ""
-        + "package test;\n"
+    JavaFileObject expected = JavaFileObjects.forSourceString("test/AutoValue_Test", "package "
+        + "test;\n"
```

Comment:
nit: `package` should still be in this line

Reply:
my autoformat keeps taking that elsewhere, will leave it for now to avoid churn